### ISSUE TITLE
[BUG] Undefined variable $className causing dd() when visiting /prequel

### ISSUE THREAD

**Issue by caendesilva** [Protoqol/Prequel#165]
**Describe the bug**
Just installed Prequel and went to /prequel, only to be greeted with this:
```
ErrorException {#559 ▼ // vendor/protoqol/prequel/src/Http/Middleware/Authorised.php:74
  #message: "Undefined variable $className"
  #code: 0
  #file: "
vendor/protoqol/prequel/
src/Connection/DatabaseConnector.php"
  #line: 32
  #severity: E_WARNING
  trace: {▶}
}
```

**To Reproduce**

1. Follow readme install instructions
2. Visit /prequel

**PHP Version**

- 8.3

**Database type**

- Sqlite3

**Comment by caendesilva** [comment]
Oh, I went further into the stacktrace and figured out that you don't support SQLite. https://github.com/Protoqol/Prequel/issues/46

**Comment by PhillipMwaniki** [comment]
I have the same issue while using mysql

**Comment by PhillipMwaniki** [comment]
**Update**
Here is a fix https://github.com/Protoqol/Prequel/pull/166

**Comment by QuintenJustus** [comment]
Fixed by #166, update to 1.23.13
